docs/2.9.0/docs/daml/intro/13_Interfaces.rst

```diff
@@ -1,8 +1,8 @@
 .. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-Daml Interfaces
-===============
+Interfaces
+==========
```

Comment:
"Daml Interfaces" is more information

Reply:
Since an interface is something to be defined like a template, function, or a custom record type, wouldn't it be better to call it "Interface" instead of "Daml Interface"? It's like calling a template "Daml Template" a function each time.
![Screenshot 2024-05-02 at 10 20 14 AM](https://github.com/digital-asset/docs.daml.com/assets/126821560/495da2a7-6963-4411-8032-e2b2eb7ac9d6)